demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -218,15 +218,19 @@ def _search_packs_by_items_names(items_names: Union[str, list],
     @staticmethod
     def _search_packs_by_items_names_or_ids(items_names: Union[str, list],
                                             items_list: list,
-                                            exclude_ignored_dependencies: bool = True) -> set:
+                                            exclude_ignored_dependencies: bool = True,
+                                            incident_or_indicator: Optional[str] = 'both') -> set:
```

Comment:
I think you are not passing this arg the calls in `_collect_layouts_dependencies` so either you should pass it or this arg is useless and should be removed

Reply:
I added it